baseplate/clients/redis_cluster.py

```diff
@@ -346,9 +347,33 @@ class ClusterRedisContextFactory(ContextFactory):
     :param connection_pool: A connection pool.
     """
 
-    def __init__(self, connection_pool: rediscluster.ClusterConnectionPool):
+    PROM_PREFIX = "bp_redis_cluster_pool"
+    PROM_LABELS = ["pool"]
+
+    promTotalConnections = Gauge(
```

Comment:
Python best practices are snake case for var names

I'd also consider dropping prom from here to match the patterns you established above (change the below as well)

Reply:
Fixed